### PR TITLE
Edits to Migration Portal

### PR DESCRIPTION
**Pull request by ebgitelman**
## What Changed?

Please list, at a high level, what changed in your branch. If you changed content, include the product, section, and version as applicable. **Please remove the user instructions including the examples before merging the PR.**

**Examples**

- Fixed typo in EPAS 13 epas_inst_linux guide
- Added more detail to ARK 3.5 getting started guide introduction
- Fixed broken link on `/supported-open-source/pgbackrest/05-retention_policy/` page

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [ ] This PR adds new content
- [ ] This PR changes existing content
- [ ] This PR removes existing content
- [ ] This PR is for a release, please add this tag:

### CODE REVIEW

**Review by drothery-edb — APPROVED** [review]
LGTM

File: product_docs/docs/migration_portal/3.4.0/04_mp_migrating_database/01_mp_schema_extraction.mdx [hunk 1]
```diff
@@ -9,27 +9,27 @@ legacyRedirectsGenerated:
...
+2.  Provide the schema name and the path or directory in which the extractor stores the extracted DDL. When extracting multiple schemas, use a comma (`‘,’`) as a delimiter.
 
 !!!Note
-    If you want to extract all the user schemas from the current database, do not mention any schema names while extracting. However, it is recommended to mention the schema names that you would like to extract.
+    If you want to extract all the user schemas from the current database, don't mention any schema names while extracting. However, we recommend that you mention the schema names that you want to extract.
 
 3.  If you want to extract dependent objects from other schemas, enter `yes` or `no`.
 
-    **For example, on Linux:**
+## On Linux
```
Comment (moiznalwalla) [inline product_docs/docs/migration_portal/3.4.0/04_mp_migrating_database/01_mp_schema_extraction.mdx:32]: Not sure if this should be a heading?

File: product_docs/docs/migration_portal/3.4.0/04_mp_migrating_database/01_mp_schema_extraction.mdx [hunk 2]
```diff
@@ -42,7 +42,8 @@ Given schema(s) list may contain objects which are dependent on objects from oth
 
 Extract dependent object from other schemas?(yes/no) (Default no / Ignored for all schemas option): yes
 ```
-**On Windows:**
+
+## On Windows
```
Comment (moiznalwalla) [inline product_docs/docs/migration_portal/3.4.0/04_mp_migrating_database/01_mp_schema_extraction.mdx:46]: Same as above.

File: product_docs/docs/migration_portal/3.4.0/05_mp_advanced_data_migration.mdx [hunk 3]
```diff
@@ -9,12 +9,9 @@ legacyRedirectsGenerated:
 
 <div id="mp_advanced_data_migration" class="registered_link"></div>
 
-For larger databases that require a parallel data load, you can use one of the following methods:
+For larger databases that require a parallel data load, you can use one of the following:
```
Comment (moiznalwalla) [inline product_docs/docs/migration_portal/3.4.0/05_mp_advanced_data_migration.mdx:12]: Feels like the word "methods" added some value in the lead in sentence.

File: product_docs/docs/migration_portal/3.4.0/index.mdx [hunk 4]
```diff
@@ -19,13 +19,24 @@ legacyRedirectsGenerated:
...
+-  Indexes
+
+The migration produces DDLs compatible with EDB Postgres Advanced Server.
 
 The user-friendly portal interface simplifies the assessment and migration process. Log in to the portal and start with the migration process.
 
-The Migration Portal guide provides a high-level description of the steps involved in the migration process. The guide also includes solutions to common migration problems and details of unsupported features and their potential workarounds.
-
-EnterpriseDB has helped companies migrate their existing database systems to Postgres for years. For more information, visit the EnterpriseDB website at: <https://www.enterprisedb.com/>
+EnterpriseDB has helped companies migrate their existing database systems to Postgres for years. For more information, see the (EnterpriseDB website)[https://www.enterprisedb.com/].
```
Comment (moiznalwalla) [inline product_docs/docs/migration_portal/3.4.0/index.mdx:39]: Fixed the broken link here while i was working on something else.

File: product_docs/docs/migration_portal/3.4.0/04_mp_migrating_database/01_mp_schema_extraction.mdx (hunk 1 above)
Comment (ebgitelman) [reply product_docs/docs/migration_portal/3.4.0/04_mp_migrating_database/01_mp_schema_extraction.mdx:32]: Thanks. Dee Dee and I looked at this together and came up with a solution.

File: product_docs/docs/migration_portal/3.4.0/04_mp_migrating_database/01_mp_schema_extraction.mdx (hunk 2 above)
Comment (ebgitelman) [reply product_docs/docs/migration_portal/3.4.0/04_mp_migrating_database/01_mp_schema_extraction.mdx:46]: See above

File: product_docs/docs/migration_portal/3.4.0/05_mp_advanced_data_migration.mdx (hunk 3 above)
Comment (ebgitelman) [reply product_docs/docs/migration_portal/3.4.0/05_mp_advanced_data_migration.mdx:12]: I think it's fine either way but I added it back,

File: product_docs/docs/migration_portal/3.4.0/index.mdx (hunk 4 above)
Comment (ebgitelman) [reply product_docs/docs/migration_portal/3.4.0/index.mdx:39]: Thanks.

**Pull request merged**